docs/standard/library.md

```diff
@@ -1,105 +1,124 @@
 ---
-title: .NET Standard Library
-description: .NET Standard Library
-keywords: .NET, .NET Core
+title: .NET Standard | Microsoft Docs
+description: Learn what is .NET Standard, its versions and the .NET platforms supported.
+keywords: .NET Standard, PCL, .NET
 author: richlander
 ms.author: mairaw
-ms.date: 06/20/2016
+ms.date: 03/16/2017
 ms.topic: article
 ms.prod: .net
 ms.technology: dotnet-standard
 ms.devlang: dotnet
 ms.assetid: c044882c-af15-45f2-96d1-534557a5ee9b
 ---
 
-# .NET Standard Library
```

Comment:
While I don't mind I'm curious why the change from ".NET Standard Library" to ".NET Standard" throughout the document? Also the document name itself is called "library.md" :)